specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -0,0 +1,2237 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Bot Service",
+    "description": "Azure Bot Service is a platform for creating smart conversational agents.",
+    "version": "2018-07-12"
```

Comment:
If this is preview, please add a -preview suffix to the api-version

Reply:
@ravbhatnagar I already updated manifest and our PPE environment with this version name. Would it be acceptable to leave it as-is, given that it is in the preview folder? If not, I can rev all the changes but it is a bit hard to coordinate.

Reply:
@carlosscastro - just to clarify - the "preview" directory in the repo is not related to the "preview" status of the API. The directory name is only to indicate the status of the swagger spec and it's stability for producing SDKs.

Reply:
@annatisch thanks for clarifying. Regardless, given that our previous API version has no '-preview' and that we already deployed some code with this API version, can we agree to leave it as-is? We will still support the previous API version simultaneously for 8+ months.

Reply:
@ravbhatnagar can correct me if I'm wrong - but is this new API version considered a stable API deployment? Or a preview API? If it's a preview API (e.g. limited availability, trial features etc) then the API version should probably have a suffix. 
However if this is a stable API and we are discussing this because of the PR title and the location in the "preview" directory then the suffix probably isn't necessary.

Though will leave this call up to @ravbhatnagar as I'm not familiar with correct process here :)

Reply:
Sure. @ravbhatnagar let us know if we can merge with API version as-is. We signed up as preview just because they are the first couple of versions, but we will be respecting all the stable requirements, i.e. long term support of past version, etc.

Reply:
Thanks!!! @annatisch can we merge this PR?

Reply:
Thanks @carlosscastro - I will do a final review today.